src/modules/did/did.js

```diff
@@ -731,12 +732,12 @@ class DIDModule {
    * @return {Promise<object>} The DID document.
    */
   // eslint-disable-next-line sonarjs/cognitive-complexity
-  async getDocument(did) {
+  async getDocument(did, { getBbsPlusSigKeys = false } = {}) {
     const hexId = getHexIdentifierFromDID(did);
 
     let didDetails = await this.api.rpc.core_mods.didDetails(hexId, 15);
     if (didDetails.isNone) {
-      throw new NoDIDError(`dock:did:${did}`);
+      throw new NoDIDError(`dock:did:${hexDIDToQualified(hexId)}`);
```

Comment:
`dock:did:` should be `did:dock:` ?

Reply:
The argument `did` could be fully-qualified or just hex so can't rely on that.

Reply:
not sure what you mean, im referring to the error message

Reply:
see the edit you made, yeah im aware after a quick search that dock:did is used - could still patch here too

Reply:
Done.